lib/crypto-primitives/crypto-primitives.cabal

```diff
@@ -25,6 +25,7 @@ library
 
   hs-source-dirs:     src
   exposed-modules:    Cryptography.Cipher.ChaChaPoly1305
+                      Cryptography.Cipher.AES256CBC
```

Comment:
I just noticed that the test suite for this is in another package (`unit.cabal`).

I propose making a dedicated test suite for `crypto-primitives`, what do you think?

I made a commit that does this here: 2c1a37045e4c177656c4b044e2e0b0152dc8c635

Feel free to cherry-pick!